### ISSUE TITLE
The last main menu entries can't be accessed on Android in landscape mode

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/extension-workshop#224]
### Describe the problem and steps to reproduce it:

1. Open the extension workshop on an Android device - https://extensionworkshop-dev.allizom.org/
2. Tap on the hamburger menu to see all the available options
3. Try scrolling down to access the last menu items

### What happened?
There is no scrolling option in the menu which makes it impossible to access the last items when the device is in landscape mode

### What did you expect to happen?
All menu items should be accessible when using landscape mode on an Android device

### Anything else we should know?
- reproduced on Android 8.0 with Fx68

![2019-07-25_15-13-29](https://user-images.githubusercontent.com/31961530/61873962-cf4f2c80-aeef-11e9-8df6-47ee0bc4582c.gif)

**Comment by caitmuenster** [comment]
@lancecummings, could you look into this?

**Comment by caitmuenster** [comment]
Note for QA: I accidentally merged this before reviewing; verification needed.

**Comment by AlexandraMoga** [comment]
@caitmuenster this issue seems fixed:

![2019-07-30_16-30-21](https://user-images.githubusercontent.com/31961530/62133410-63056c00-b2e7-11e9-93cf-31137ff8beeb.gif)

**Comment by caitmuenster** [comment]
Yay, thank you @AlexandraMoga!
